src/store/mutations.ts

```diff
@@ -30,7 +30,12 @@ type DomainsMutation = {
 
 type PipelineMutation = {
   type: 'setPipeline';
-  payload: Pick<VQBState, 'pipeline'>;
+  payload: Pipeline;
```

Comment:
Shouldn't this be `payload: { pipeline: Pipeline }` to be consistent with other mutations?

Reply:
What I want to express here is: it should be the type of any of the properties values of pipelines.
We may be able to describe thas as `Pick<<Pick<VQBState, 'pipeline'>, any>` but:
- I'm unsure the `any` as second parameter will match any of the pipeline's keys
- it's really confusing to read
- if someone puts something of the wrong type in state.pipelines[...], it would authorize something wrong here.

So I really prefer to keep that type simple and readable: it should be a `Pipeline`